modules/configuring-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,127 @@
+//* installing-restricted-networks-installer-provisioned-vsphere.adoc [IPI]
+//* installing-restricted-networks-vsphere.adoc [IPI]
+
+:_content-type: PROCEDURE
+[id="configuring-vsphere-regions-zones_{context}"]
+= Configuring regions and zones for a VMware vCenter
+You can modify the default installation configuration file, so that you can deploy an {product-title} cluster to multiple vSphere datacenters that run in a single VMware vCenter. 
+
+[NOTE]
+====
+The default `install-config.yaml` file configuration from the previous release of {product-title} is deprecated. You can continue to use the deprecated default configuration, but the `openshift-installer` will prompt you with a warning message that indicates the use of deprecated fields in the configuration file. 
+====
+
+[IMPORTANT]
+====
+The example uses the `govc` command. The `govc` command is an open-source command available from VMware; it is not available from Red Hat. The Red Hat support team does not maintain the `govc` command. Instructions for downloading and installing `govc` are found on the VMware documentation website
+====
+
+.Prerequisites
+* You have an existing `install-config.yaml` installation configuration file.
+
+.Procedure
+
+. Use the `govc` command-line tool to create vCenter tag categories. The following example generates tags for zones and for regions: 
++
+[source,terminal]
+----
+$ govc tags.category.create -d "<region_tag_category_name>" <region_tag_category>
+
+$ govc tags.category.create -d "<zone_tag_category_name>" <zone_tag_category>
+----
+
+. Create a region tag for each region:
++
+[source,terminal]
+----
+$ govc tags.create -c <region_tag_category> <region_tag>
+----
+
+. Create a zone tag for each zone:
++
+[source,terminal]
+----
+$ govc tags.create -c <zone_tag_category> <zone_tag>
+----
+
+. Attach region tags to each vCenter datacenter object: 
++
+[source,terminal]
+----
+$ govc tags.attach -c <region_tag_category> <region_tag_1> /<datacenter_1>
+----
+
+. Attach the zone tags to each vCenter datacenter object:
++
+[source,terminal]
+----
+$ govc tags.attach -c <zone_tag_category> <zone_tag_1> /<datacenter_1>/host/vcs-mdcnc-workload-1
+----
+
+. Create the `install-config.yaml` file and configure the file to support operating multiple datacenters in a vSphere center.
++
+[IMPORTANT]
+====
+You must specify at least one failure domain for your {product-title} cluster, so that you can provision datacenter objects for your VMware vCenter server. Consider specifying multiple failure domains if you need to provision virtual machine nodes in different datacenters, clusters, datastores, and so on.
+====
+
+. Change to the directory that contains the installation program and initialize the cluster deployment according to your chosen installation requirements.
+
+.Sample `install-config.yaml` file with multiple datacenters defined in a vSphere center
+
+[source,yaml]
+----
+...
+compute:
+...
+  vsphere:
+      zones:
+        - "<zones_tag_1>"
```

Comment:
I still wonder if this should be more clear that a machine pool zone item must be listed as a string in failureDomain name?

Reply:
Updated example with the following strings. WDYT?

![Uploading Screenshot from 2023-03-06 17-00-59.png…]()

Reply:
I think that is fine as long as this...

The list of MachinePool zones must be a subset (or equal) to the list of failureDomain names.

failureDomain[*].Name defines the available MachinePool zones that are possible.

```
controlPlane:
...
vsphere:
     zones:
       - "<machine_pool_zone_1>"
       - "<machine_pool_zone_2>"
platform:
  vsphere:
    failureDomain:
    - name: "<machine_pool_zone_1>"
```

Reply:
Thanks. For the procedure example, the MP zones match the list of FD names in the `Configuring regions and zones for a VMware vCenter` file example. The vanilla sample YAML file does not go into this level of detail, so I have not established MP zones and listed FDs.